web/views/recurring_jobs.erb

```diff
@@ -1,50 +1,74 @@
 <h3><%= t('recurring_jobs') %></h3>
 
-<div class="table_container">
-  <table class="table table-hover table-bordered table-striped table-white">
-    <thead>
-      <tr>
-        <th><%= t('name') %></th>
-        <th><%= t('description') %></th>
-        <th><%= t('interval') %></th>
-        <th><%= t('class') %></th>
-        <th><%= t('queue') %></th>
-        <th><%= t('arguments') %></th>
-        <th><%= t('last_time') %></th>
-        <th><%= t('next_time') %></th>
-        <th></th>
-      </tr>
-    </thead>
+<style type="text/css">
+  .recurring-jobs { border-top-left-radius: 4px; border-top-right-radius: 4px; }
+  .recurring-jobs .title { margin-bottom: 5px; }
+  .recurring-jobs .title .name { font-weight: bold;}
+  .recurring-jobs .info,
+  .recurring-jobs .description { margin-bottom: 5px; }
+  .recurring-jobs .actions { margin-bottom: 5px; }
+  .recurring-jobs .status,
+  .recurring-jobs .description { font-size: 12px; }
+  .recurring-jobs .enqueue { margin-bottom: 0.5rem }
 
-    <tbody>
-      <% @presented_jobs.each do |job| %>
-        <tr>
-          <td><%= job.name %></td>
-          <td><%= job['description'] %></td>
-          <td><%= job.interval %></td>
-          <td><%= job['class'] %></td>
-          <td>
-            <a href="<%= root_path %>queues/<%= job.queue %>"><%= job.queue %></a>
-          </td>
-          <td><%= job['args'] %></td>
-          <td><%= job.last_time %></td>
-          <td>
-            <span style="<%= 'text-decoration:line-through' unless job.enabled? %>">
-              <%= job.next_time || t('no_next_time') %>
-            </span>
-          </td>
-          <td class="text-center">
-            <form action="<%= root_path %>recurring-jobs/<%= ERB::Util.url_encode(job.name) %>/enqueue" method="post">
+  .list-group-item {
+    background-color: #222;
+    color: white;
+    border: 1px solid #555;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    .list-group-item {
+      background-color: #222;
+      color: white;
+      border: 1px solid #555;
+    }
+  }
+</style>
```

Comment:
is there a reason why this is embedded in the html file instead of a separate css?

Reply:
Not a specific reason, we can move it to a css file without problems

Reply:
because there is nowebpacker to precompile the scss/css I don't see keeping it in the html file would be bad. However because this file slimmer, I think keeping styles in a sepparate css file would be the best option. Is there a way to insert the `<link rel="stylesheet" href="styles.css">` into the head from here using `content_for(:head)`?

Reply:
what are your thoughts on this?

Reply:
Yes I think that would be a better option, tomorrow I'll take a look on how sidekiq does it push some changes

Reply:
@juan-apa Using `content_for` won't work because sidekiq and sidekiq-scheduler are framework agnostic, and `content_for(:head)` is an ActionView functionality. 

I've moved the css to an specific file and added Rack::Static as a middleware to serve this file

Reply:
yep, you are right, got so used to some of rails helpers that I assumed they were always there 😅.